seismic/inversion/sandbox/synth_rjmcmc_validation/synth_2layer.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
New file: incidental work done at the same time to produce synthetic test data for RF inversion testing.